python/doc/examples/numerical_methods/optimization/plot_optimization_pagmo.py

```diff
@@ -0,0 +1,96 @@
+"""
+Multi-objective optimization using Pagmo
+========================================
+"""
+# %%
+# In this example we are going to explore optimization using OpenTURNS' `Pagmo <https://esa.github.io/pagmo2/>`_ interface.
+
+# %%
+from __future__ import print_function
+import openturns as ot
+from openturns.viewer import View
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# List available algorithms
+for algo in ot.Pagmo.GetAlgorithmNames():
+    print(algo)
+
+# %%
+# More details on Pagmo algorithms are available `here <https://esa.github.io/pagmo2/overview.html#list-of-algorithms>`_ .
+
+# %%
+# Create the problem, from ZDT test suite
+f = ot.SymbolicFunction(['x1', 'x2'], ['x1', 'var s := x1 + x2; var g := 1 + 9 * s; g * (1.0 - sqrt(x1 / g))'])
+zdt1 = ot.OptimizationProblem(f)
+zdt1.setBounds(ot.Interval([0.0] * 2, [1.0] * 2))
+
+# %%
+# Create the first generation of points
+pop0 = ot.ComposedDistribution([ot.Uniform(0.0, 1.0)] * 2).getSample(100)
```

Comment:
The sample size is too large to get a visually interesting picture. 

Reducing the size from 100 to 10 should move your picture from:

![image](https://user-images.githubusercontent.com/31351465/155683939-b958863a-06ff-4a8a-9ad5-4e2de5f7ea1d.png)

to the picture presented in Pygmo:

![image](https://user-images.githubusercontent.com/31351465/155683909-7f6d4934-0be5-4e89-a23f-07c3ed8470fd.png)

Moreover, adding bullets to the break points (as in Pygmo's doc) may increase the visual impact.

Reply:
actually its way worse